modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update. This can be useful to determine if history-related bugs apply to your cluster. The entry can also reveal correlation between changes in cluster behavior with potential triggers, although correlation does not imply causation. 
+
+[NOTE]
+====
+The ClusterVersion history has a size limit, so in cases where enough updates occurred to have reached that limit, the least relevant entries in the history are pruned to accommodate the limit. 
+====
+
+It is possible to view the ClusterVersion history though the web console and through the CLI.
+
+To view the ClusterVersion history through the web console:
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. From the web console, click *Administration* -> *Cluster Settings* and review the contents of the *Details* tab.
+
+To view the ClusterVersion history through the CLI:
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. View cluster update history:
++
+[source,terminal]
+----
+$ oc describe clusterversions/version
+----
++
+.Example output
+[source,terminal]
+----
+  $ oc describe clusterversions/version
+...
```

Comment:
Do not include the command in the example output code block:

![image](https://github.com/openshift/openshift-docs/assets/40172997/74798f97-91b7-49a1-ac24-045ef080cd2a)

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output